consensus/consensus-types/src/block.rs

```diff
@@ -264,6 +266,31 @@ impl Block {
                 "Reconfiguration suffix should not carry payload"
             );
         }
+        if let Some(failed_authors) = self.block_data().failed_authors() {
+            // when validating for being well formed,
+            // allow for missing failed authors,
+            // for whatever reason (from different max configuration, etc),
+            // but don't allow anything that shouldn't be there.
+            //
+            // we validate the full correctness of this field in round_manager.process_proposal()
+            let skipped_rounds = self
+                .round()
+                .saturating_sub(parent.round())
```

Comment:
for verification, we probably should use checked_sub and return error when overflow